src/modules/user/application/use_cases/update_security_use_case.py

```
@@ -66,22 +71,50 @@ async def execute(self, user_id: str, request: UpdateSecurityRequestDTO) -> Upda
+            try:
+                email_sent = email_service.send_verification_email(
+                    to_email=request.new_email,
+                    user_name=user.first_name,
+                    verification_token=verification_token
+                )
+                if not email_sent:
+                    logger.warning(
+                        "No se pudo enviar el email de verificación para el usuario %s",
+                        user.id.value
+                    )
+            except (requests.RequestException, ValueError, ConnectionError) as e:
```

Ruff (0.14.4)

Undefined name `requests`

(F821)